packages/telegram-stamper/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# @turnkey/telegram-stamper
```

Comment:
to be future proof I wonder if we should name this "telegram-cloud-storage-stamper"? I'm thinking we might have a "telegram-biometrics-stamper" down the road.

Reply:
changed to telegram-cloud-storage-stamper